src/main/resources/org/jenkinsci/test/acceptance/docker/fixtures/SvnContainer/Dockerfile

```diff
@@ -1,75 +1,59 @@
 # Sets up
-FROM ubuntu:precise
-RUN apt-get update && apt-get install -y wget
-
-# Install Subversion 1.8 and Apache
-RUN echo 'deb http://archive.ubuntu.com/ubuntu precise main universe' >> /etc/apt/sources.list
-RUN sh -c 'echo "deb http://opensource.wandisco.com/ubuntu precise svn18" >> /etc/apt/sources.list.d/WANdisco.list'
-RUN wget -q http://opensource.wandisco.com/wandisco-debian.gpg -O- | apt-key add -
-RUN apt-get update -y
-RUN apt-get install -y subversion apache2 libapache2-svn
+FROM debian:buster
```

Comment:
ubuntu has no package for `viewvc` or `viewsvn`.  We do not actually use this currently and I thought about removing it, but for now I decided to leave it in.